apps/bfd-model/bfd-model-rif/src/main/resources/db/scripts/Query_synthetic_id_ranges.sql

```diff
@@ -0,0 +1,102 @@
+-- Find the beneficiaries.bene_id range to avoid collisions with synthetic data.
+WITH ids AS (
+  SELECT
+    CAST(bene_id AS numeric) AS id
+  FROM beneficiaries
+  ORDER BY CAST(bene_id AS numeric)
+),
+groupings AS (
+  SELECT
+    ROW_NUMBER() OVER (ORDER BY id) - id AS grouping,
+    id
+  FROM ids
+)
+SELECT MIN(id) AS grouping_start,
+       MAX(id) AS grouping_end
+FROM groupings
+GROUP BY grouping
+ORDER BY MIN(id);
+
+-- Find  min claim Ids for all claims except Part D Events.
+-- This serves as an end range to avoid collisions with future synthetic data batches.
```

Comment:
I think it should also be said that new synthetic data should start with clm_id's that are one less than the value returned by this query